pkg/specs/jobs.go

```diff
@@ -318,6 +318,7 @@ func (role jobRole) getJobName(instanceName string) string {
 func CreatePrimaryJob(cluster apiv1.Cluster, nodeSerial int, role jobRole, initCommand []string) *batchv1.Job {
 	instanceName := GetInstanceName(cluster.Name, nodeSerial)
 	jobName := role.getJobName(instanceName)
+	version, _ := cluster.GetPostgresqlMajorVersion()
```

Comment:
I would add at least the minor version. What do you think @JefeDavis ?

Reply:
Is the minor version available anywhere? I couldn't, from what I saw only the major version is recorded and available for the cluster